scripts/cibuild.rb

```diff
@@ -95,12 +95,6 @@ def verify_file (f)
       return [f, error]
     end
 
-    if dups.any? then
```

Comment:
@ramsanath can you please define what these changes are?

Reply:
@ritwik12 I noticed that the code to check for duplicate tags was present twice.
So I removed the unnecessary error check.
<img width="495" alt="Screenshot 2019-08-22 at 1 13 28 PM" src="https://user-images.githubusercontent.com/25882182/63496025-b893f980-c4de-11e9-81db-4cc097aae29e.png">